app/src/main/kotlin/com/simplemobiletools/flashlight/activities/MainActivity.kt

```diff
@@ -300,7 +304,7 @@ class MainActivity : SimpleActivity() {
     }
 
     private fun changeIconColor(color: Int, imageView: ImageView?) {
-        imageView!!.background.mutate().applyColorFilter(color)
```

Comment:
no need to call mutate here as applyColorFilter automatically call mutate() on the same drawable